projects/addon-doc/components/example/example.style.less

```diff
@@ -56,6 +56,8 @@
 
 .t-demo {
     .customize-scroll();
+    display: flex;
+    align-items: flex-start;
```

Comment:
Many example has problems with layout now :(

For example: https://taiga-ui--pr4905-e2e-size-p44bcl8e.web.app/components/input-number#sizes
Initially: https://taiga-ui.dev/components/input-number#sizes

Reply:
Oh, I forgot `flex-direction`. Still broken though, I'll think about it.